cactusbot/handlers/events.py

```diff
@@ -23,8 +26,20 @@ def on_follow_return():
 
         if packet.success:
             if self.cache_follows:
-                if not(CacheUtils.in_cache_followers(packet.user)):
-                    CacheUtils.cache_followers_add(packet.user)
+                cache = CacheUtils("caches/followers.json")
```

Comment:
Initializing a new instance of `CacheUtils` every time someone follows is pretty "clunky". Instead, maybe move this to `__init__` and use `self.cache`?